[2024-12-19T07:18:03.929+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T07:18:03.948+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-15T00:00:00+00:00 [queued]>
[2024-12-19T07:18:03.953+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-15T00:00:00+00:00 [queued]>
[2024-12-19T07:18:03.954+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T07:18:03.965+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): second_task> on 2021-09-15 00:00:00+00:00
[2024-12-19T07:18:03.971+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=8883) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T07:18:03.972+0000] {standard_task_runner.py:72} INFO - Started process 8904 to run task
[2024-12-19T07:18:03.974+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'scheduled__2021-09-15T00:00:00+00:00', '--job-id', '1623', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpf3m0amsz']
[2024-12-19T07:18:03.978+0000] {standard_task_runner.py:105} INFO - Job 1623: Subtask second_task
[2024-12-19T07:18:04.021+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.second_task scheduled__2021-09-15T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T07:18:04.074+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2021-09-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-09-15T00:00:00+00:00'
[2024-12-19T07:18:04.074+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T07:18:04.087+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T07:18:04.088+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1']
[2024-12-19T07:18:04.094+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T07:18:04.095+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1
[2024-12-19T07:18:04.096+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T07:18:04.113+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T07:18:04.114+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=scheduled__2021-09-15T00:00:00+00:00, execution_date=20210915T000000, start_date=20241219T071803, end_date=20241219T071804
[2024-12-19T07:18:04.157+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T07:18:04.165+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T07:18:04.166+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
